packages/create-react-native-library/src/index.ts

```diff
@@ -319,7 +321,7 @@ async function create(argv: yargs.Arguments<any>) {
       : 'module';
 
   // Get latest version of Bob from NPM
-  let version: string;
+  let version: string = FALLBACK_BOB_VERSION;
```

Comment:
Instead of this, let's do `version: version || FALLBACK_BOB_VERSION` below where it's used. it'll make sure it works even if `version` is empty for some reason